reconcile/aus/base.py

```diff
@@ -684,6 +708,32 @@ def calculate_diff(
                         ),
                     )
                 )
+            elif (
+                cluster.spec.hypershift is not None and cluster.spec.hypershift is True
```

Comment:
This could just be 
```suggestion
                cluster.spec.hypershift
```
But I think we may have a `None` `spec` for imported clusters, so maybe a check like `cluster.spec is not None` is needed ?

Reply:
I'll revisit this, your proposal was breaking tests. Perhaps I can find another way to fix them